README.md

```diff
@@ -360,11 +360,10 @@ script:
   - ./guard --exit-code 1 guard-ci-test:${COMMIT}
 ```
 
-Example: https://travis-ci.org/goodwithtech/guard-ci-test
-
+Example: https://travis-ci.org/goodwithtech/guard-ci-test<br/>
 Repository: https://github.com/goodwithtech/guard-ci-test
 
-## Circle CI
+## CircleCI
```

Comment:
Please change `[Circle CI](#circle-ci)` to  [CircleCI](#circleci)  in TOC.

Reply:
I fixed it.